receiver/dockerstatsreceiver/receiver.go

```diff
@@ -264,3 +266,14 @@ func (r *receiver) recordPidsMetrics(now pcommon.Timestamp, pidsStats *dtypes.Pi
 		}
 	}
 }
+
+func (r *receiver) recordBaseMetrics(now pcommon.Timestamp, base *types.ContainerJSONBase) {
+	t, err := time.Parse(time.RFC3339, base.State.StartedAt)
+	if err != nil {
+		// value not available or invalid
```

Comment:
This sounds like something that should be bubbled up as a [partial scraper error](https://pkg.go.dev/go.opentelemetry.io/collector/receiver/scrapererror#PartialScrapeError)

Reply:
Done.